tools/wutool/wutool.cpp

```diff
@@ -49,8 +49,8 @@ void usage()
            "   delete <workunits>  - Delete workunits\n"
            "   results <workunits> - Dump results from specified workunits\n"
            "\n"
-           "   archive <workunits> - Archive to xml files [TO=<directory>] [DEL=1] [KEEPFILERESULTS=1]\n"
-           "   restore <filenames> - Restore from xml files\n"
+           "   archive <workunits> - Archive to xml files [TO=<directory>] [DEL=1] [KEEPFILERESULTS=1] [INCLUDEFILES=1]\n"
```

Comment:
What are the numbers following the ids meant to be?  the defaults, or the values to override it.
It looks like del defaults to false, keepfileresults to true.